invoices_app/tests.py

```diff
@@ -47,8 +47,8 @@ def setUp(self):
         self.file_mock.size = 50
 
     def tearDown(self):
-        if path.exists(self.file_mock.name):
-            remove(self.file_mock.name)
+        if(self.file_mock and path.exists('file/' + self.file_mock.name)):
```

Comment:
there is no need for parenthesis, (use it whe is a multiline if) 
`if self.file_mock and path.exists('file/{}'.format(self.file_mock.name)):`
or if you are using python3.7:
`if self.file_mock and path.exists('file/{self.file_mock.name}'):`